packages/algoliasearch-helper/index.d.ts

```diff
@@ -1326,6 +1326,49 @@ declare namespace algoliasearchHelper {
           };
         };
       };
+      /**
+       * Defining UI widget configuration
+       */
+      widgets?: {
+        /**
+         * Configuration for banners
+         */
+        banners?: Array<{
+          /**
+           * Configuration for the banner image
+           */
+          image?: Array<{
+            /**
+             * Set of possible URLs of the banner image
+             */
+            urls?: Array<{
+              /**
+               * URL of the banner image
+               */
+              url?: string;
+            }>;
+            /**
+             * Alt text of the banner image
+             */
+            title?: string;
+          }>;
```

Comment:
Shouldn't those types be required ? Does it make sense to have a banner without an image ? At least `urls`, `url` and `title` should be required.

Reply:
Yes, @aymeric-giraudet - you're correct. I will update that now.

Reply:
I left `title` optional since that's what product discussed in their specification document and what was left in the RFC. `image`, `urls`, and `url` are all required though (because like you said: there is no point in having a banner with the banner image url).